php-frontend/src/test/java/org/sonar/plugins/php/api/tests/package-info.java

```diff
@@ -1,21 +0,0 @@
-/*
```

Comment:
@vilchik-elena compilation was failing, so I removed it. It turns out that is a problem with Eclipse, when the package in both src/main/java and src/test/java has that file. I could have removed it in a separate commit.